agentos/run.py

```diff
@@ -36,8 +41,19 @@ class Run:
     - Component Registry -> MLflow artifact file
     - Entry point string -> MLflow run tag (MlflowRun.data.tags entry)
     - ParameterSet -> MLflow artifact file
+
+    A Run can also contain a [pointer to a] RunCommand.
+
+    A Run can also have pointers to other runs. These pointers can have
+    different semantic meanings. They could have an "inner-outer" relationship,
+    i.e., an outer run is already active when it's "inner" run starts and ends.
+    Alternatively, two runs could have an "output-input dependency-depender"
+    relationship, in which the depender uses the some part of dependency run's
+    output as input. In this case, the relationship is causal.
     """
 
+    # TODO: Decide if pointers to other runs should be special some how.
```

Comment:
How are these pointers currently stored?  As params?

Reply:
MLflow uses tags for storing pointers to a Run's parent. The key for the tag to use is defined in `mlflow.utils.mlflow_tags.MLFLOW_PARENT_RUN_ID`. Actually that same file defines a bunch of standard MLflow tag keys.

I think for now, the only form of pointers to other runs that we manage are parents.